about.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Unique Computers</title>
+</head>
+<body>
+    <div id="container">
+        <section>
+            <div id="About">
+                <img src="" alt="About Us">
```

Comment:
remove the image tag. we will add the image directly with css

Reply:
okay